app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html

```diff
@@ -68,6 +72,34 @@
       } else None
     ))
 
-    @button("continue.button")
+
+
+} else {
+
+    @H1(messages("businessEntityEoriNew.title"), id = Some("businessEntityEoriTitleId"))
+    @P(messages("businessEntityEori.p1"), id = Some("businessEntityEoriP1Id"))
+    @P(messages("businessEntityEori.p2"), id = Some("businessEntityEoriP2Id"))
+    @P(messages("businessEntityEori.p3"), id = Some("businessEntityEoriP3Id"))
+
+
+    @govukInput(Input(
+    value = form.data.get("businessEntityEori"),
```

Comment:
try to indent/format these lines so its easier to read

Reply:
updated